### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by itslenny** [martinroob/ngx-i18nsupport#19]
I can't install this module because of the following error:

```
ENOENT: no such file or directory, chmod '...\AppData\Roaming\npm\node_modules\ngx-i18nsupport\dist\xliffmerge\xliffmerge'
```

It looks like maybe the latest npm was published without the dist folder?

**Comment by itslenny** [comment]
I just tried `npm i ngx-i18nsupport@0.2.3` and it worked so I'm guessing it's just 0.3.0 that is broken.

**Comment by martinroob** [comment]
Oops, sorry about that.
There was a compilation problem and I was a litle bit in hurry and ignored the travis output.
Should be fixed in 0.3.1 now.
